doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -2591,6 +2650,16 @@ NCSDK-18263: |NCS| samples may fail to boot on Thingy:53
 Bluetooth samples
 =================
 
+Bluetooth samples
+=================
+
```

Comment:
```suggestion
Bluetooth samples
=================

```
This seems to be duplicated

Reply:
Whoops, thanks for catching this. Fixed.